website/education/migrations/0013_auto_20200907_1835.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.1.1 on 2020-09-07 16:35
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('education', '0012_auto_20200125_1748'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='category',
+            name='name_nl',
+        ),
+        migrations.RemoveField(
+            model_name='course',
+            name='name_nl',
+        ),
+        migrations.AlterField(
+            model_name='exam',
+            name='language',
+            field=models.CharField(choices=[('en', 'English')], max_length=2, null=True),
```

Comment:
The actual language field of the documents is something we may want to keep. The documents in these models are now not suddenly translated